node/tools/README.md

```diff
@@ -0,0 +1,5 @@
+# Running a test node
+
+1. Generate a file named `config.txt` in the root directory of the tools crate, containing node addresses in the format `IP:PORT`, with each address on a separate line.
```

Comment:
nit: this is a metaconfig in a sense (used to generate configs), so perhaps giving it a different name than "config" would be helpful.

Reply:
Sounds reasonable, I'm using `addresses` now to be more specific